addons/account/models/account_move_send.py

```diff
@@ -411,7 +413,7 @@ def _hook_if_success(self, moves_data):
         to_send_mail = {
             move: move_data
             for move, move_data in moves_data.items()
-            if 'email' in move_data['sending_methods'] and self._is_applicable_to_move('email', move)
+            if 'email' in move_data['sending_methods'] and self._is_applicable_to_move('email', move, mail_partner_id=move_data['mail_partner_ids'])
```

Comment:
So in master, the plan is to pass `move_data` instead of `move` I guess

Reply:
indeed